examples/3d_examples/RestartExample/restart_example.py

```diff
@@ -0,0 +1,425 @@
+import elastica as ea
+import numpy as np
+import sopht.simulator as sps
+import sopht.utils as spu
```

Comment:
This seems to cause an ImportError. In `spu`, which includes the `restart_sim.py` file, some classes of `sps` are imported (the flow classes). Since `sps` is imported first (line 3), importing the flow classes may cause a repeated import error. Switching these two lines seem to resolve the issue.

```suggestion
import sopht.utils as spu
import sopht.simulator as sps
```